### PR TITLE
Doc: warnings on object's order and id/uri

### PR DESCRIPTION
**Pull request by pbougue**

### CODE REVIEW

File: documentation/slate/source/includes/interface.md [hunk 1]
```diff
@@ -126,3 +126,29 @@ That means :
 * where you can find the details of your object ( **"id": "edc46f3a-ad3d-11e4-a5e1-005056a44da2"** ).
 
 
+
+Objects order
+------------
+
+Unless specified otherwise objects are not sorted and stability of object's order is not guaranteed.
```
Comment (stifoon) [inline documentation/slate/source/includes/interface.md]: would prefer
"Unless specified, objects are not sorted and stability of object's order is not guaranteed."
no?

Comment (stifoon) [reply documentation/slate/source/includes/interface.md]: plurial
objects' order

File: documentation/slate/source/includes/interface.md [hunk 2]
```diff
@@ -126,3 +126,29 @@ That means :
...
+
+Examples of sorted objects:
+* journeys in a '/journeys' response
+* '/departures' and '/arrivals'
+* stop_points in '/routes/<id_route>/stop_points'
+* '/places_nearby'
+* '/places'
+
```
Comment (stifoon) [inline documentation/slate/source/includes/interface.md]: should explain the complex sort of "stop_schedules" ;)
and "traffic_reports" \o/

File: documentation/slate/source/includes/interface.md [hunk 3]
```diff
@@ -126,3 +126,29 @@ That means :
...
+* stop_points in '/lines/<id_line>/stop_points'
+* pretty much everything else...
+
+
+
+Lifetime of id
+--------------
+
+The way id (or uri) of objects are generated is not garanteed stable, nor are the data processed.
+So we advise to limit to the minimum storing id/uri of objects.
```
Comment (stifoon) [inline documentation/slate/source/includes/interface.md]: ids/uris?

File: documentation/slate/source/includes/interface.md [hunk 4]
```diff
@@ -126,3 +126,29 @@ That means :
...
+Examples of unsorted responses:
+* stop_points in '/lines/<id_line>/stop_points'
+* pretty much everything else...
+
+
+
+Lifetime of id
+--------------
+
+The way id (or uri) of objects are generated is not garanteed stable, nor are the data processed.
```
Comment (stifoon) [inline documentation/slate/source/includes/interface.md]: ids/uris

File: documentation/slate/source/includes/interface.md (hunk 2 above)
Comment (pbougue) [reply documentation/slate/source/includes/interface.md]: To me it's not the point of this part of doc... Can be in the same PR though, but I have to dig into it a bit before writing it.

**Pull request merged**